### DISCUSSION TITLE
Why are you removing vue proxy packages?

### DISCUSSION THREAD

**Discussion by mduenias** [telekom/scale#1556] [Q&A] [answered]
Hi Scale team!

I am part of a team developing a design system using Stencil and your product is a great source of inspiration to us! We were considering using the Stencil Vue output target to gain some of the features it brings out of the box including type checking. So we are curious to understand why did you decided to move away from it?

Is it only in response to Vue good support of web components, or did you experienced issues with the Vue output target?

Thanks!

**Comment by acstll** [comment, marked as answer]
Hey @mduenias — thanks for asking and sorry for our late reply. Also thanks for the kind words 💜

> Is it only in response to Vue good support of web components, or did you experienced issues with the Vue output target?

it's basically both. Vue support for custom elements is great, and we've had some trouble with "output target" plugin/packages in general. The balance between the trouble they cause and the benefits they provide is definitely not good.

Type checking and other nice stuff related to DX are definitely lost but I'm pretty sure (mostly hopeful) there are ways to make it a great experience without any wrappers.

**Comment by mduenias** [comment]
Hi! 
Thanks so much for your reply! We decided not to go with the vue output target because we found  that we could provide some configuration directly to the IDE's to support DX, at least for Vue consumers.

Sharing this in case you or anybody else is interested:
**VSCode:** [VScode Custom Data](https://github.com/microsoft/vscode-custom-data/tree/main/samples/webcomponents). Works with stencil [VSCode documentation](https://stenciljs.com/docs/docs-vscode) out of the box, very easy to configure.
**JetBrains:** [Web-Types](https://github.com/JetBrains/web-types) It needs a bit of scripting to generate the metadata, but works very well for JetBrains IDE's. They provide good examples to generate the json file and have one [implementation in ionic](https://github.com/ionic-team/ionic-framework/pull/22428) where we got inspiration from.

Thanks again!

**Reply by acstll** [reply]
Amazing, thanks for the feedback, that's actually pretty useful!
